portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +27,28 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> messages;
+
+  @Override
```

Comment:
You can initialize the messages at declaration time. This way you can remove the init function, and make variable static final.

private static final ImmutableList<String> MESSAGES = ImmutableList.of(
"I hope your day is going well!",
    "Welcome to my website :)",
   "Hope you are doing well today!"
);